packages/web-app-external/src/App.vue

```diff
@@ -111,10 +112,12 @@ const loadAppUrl = useTask(function* (signal, viewMode: string) {
 
     const fileId = props.resource.fileId
     const baseUrl = urlJoin(configStore.serverUrl, capabilityStore.filesAppProviders[0].open_url)
+    const isOnlyOfficeApp = unref(appName)?.toLowerCase().includes('onlyoffice')
```

Comment:
Please, sync with @2403905 if this should be hardcoded or if the app registry will somehow expose whether the app has a mobile view. @mzner or do you think we should always send it and leave it out to the app to either ignore it or use it?